Resync `svg` from WPT upstream

Ahmad-S792 opened the pull request:
#### b738e4236868cc1f1696689818399011c288b334
<pre>
Resync `svg` from WPT upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=296359">https://bugs.webkit.org/show_bug.cgi?id=296359</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/26b155ed94e1687af94e010c71ca0febe4c5346e">https://github.com/web-platform-tests/wpt/commit/26b155ed94e1687af94e010c71ca0febe4c5346e</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/animateMotion-keyPoints-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/animateMotion-keyPoints-002.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/crashtests/firefox-bug-1974334.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/end-of-time-001-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/end-of-time-002-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keypoints-attribute-trailing-semi-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keypoints-attribute-trailing-semi.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keytimes-attribute-trailing-semi-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keytimes-attribute-trailing-semi.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/coordinate-systems/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/crashtests/firefox-bug-1966754.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-005-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-005.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-006-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-006.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/async-04-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/async-04.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/zero-scale-svg-transform-inside-g-tag-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/zero-scale-svg-transform-inside-g-tag.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-svg-inline-css-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-svg-inline-css.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-001.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-003-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-003.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/scripted/getcharnumatposition-slr.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/scripted/getstartpositionofchar.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/types/scripted/SVGGraphicsElement.getBBox-03-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/types/scripted/SVGGraphicsElement.getBBox-03.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/w3c-import.log:
* LayoutTests/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/297792@main">https://commits.webkit.org/297792@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/44d8e07d72d0776e5c9558cf227a3ab92b7b54cb

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/112856 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/32591 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/23069 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/119060 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/63382 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/33243 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/41154 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/85891 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/36543 "") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/115803 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/26536 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/101524 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/66195 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/25824 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/19656 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/62818 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/95930 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/19730 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/122281 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/39934 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/29779 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/94755 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/40318 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/97744 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/94493 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24121 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/39619 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/17432 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/36044 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/39820 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/39460 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/42793 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/41198 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/fe19304b1f91c2449ba884e6c92d3c1aa6c99af1. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48402

Ahmad-S792 committed:
Resync `svg` from WPT upstream
https://bugs.webkit.org/show_bug.cgi?id=296359

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/26b155ed94e1687af94e010c71ca0febe4c5346e

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/animateMotion-keyPoints-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/animateMotion-keyPoints-002.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/crashtests/firefox-bug-1974334.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/end-of-time-001-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/end-of-time-002-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keypoints-attribute-trailing-semi-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keypoints-attribute-trailing-semi.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keytimes-attribute-trailing-semi-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/keytimes-attribute-trailing-semi.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/coordinate-systems/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/crashtests/firefox-bug-1966754.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-005-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-005.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-006-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-006.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/async-04-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/async-04.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/interact/scripted/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/zero-scale-svg-transform-inside-g-tag-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/zero-scale-svg-transform-inside-g-tag.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-svg-inline-css-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-svg-inline-css.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-001.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-003-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/use-symbol-inline-css-003.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/struct/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/dominant-baseline-central-large-font-size.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/scripted/getcharnumatposition-slr.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/text/scripted/getstartpositionofchar.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/types/scripted/SVGGraphicsElement.getBBox-03-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/types/scripted/SVGGraphicsElement.getBBox-03.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/w3c-import.log:
* LayoutTests/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/297792@main
